deploy/docker-compose/README.md

```diff
@@ -0,0 +1,39 @@
+# Deploying Caduceator
+
+## Docker
+
+In order to deploy into Docker, make sure [Docker is installed](https://docs.docker.com/install/). You will also need to have the AWS CLI installed. If you are on Mac OSX, you can use homebrew to install it by executing this command on the terminal: `brew install awscli`
+
+#### Deploy
+
+1. Clone this repository
+
```

Comment:
Add a new step 2 to download `awscli`, which is a dependency for the dynamodb bash script.